.docker/test.ini

```diff
@@ -22,6 +22,9 @@ host = 0.0.0.0
 port = 3000
 
 [app:main]
+ckan.devserver.host = 0.0.0.0
```

Comment:
good that it is now explicit

Reply:
Actually it already was, but the property name has changed.